department-of-veterans-affairs/LEAF
# LEAF-4069-Editor-UX-2

### Summary:
Implemented modifications to the Template Editor, Email Template Editor, and LEAF Programmer, specifically focusing on the comparison sections. These changes encompass both stylistic enhancements and functional improvements.

### Potential Impact:
Users will now have the capability to merge individual sections from older files and save them in the compared section.

### Testing:
Ensure the following aspects are thoroughly tested:

- In all comparison modes, confirm that the code area spans 100vh, consistent with normal editing mode.

- Eliminate the additional vertical scrollbar on the right-hand side in all comparison modes, addressing potential border spacing differences.

- Replace the "Restore Original" option with a "Save" button in comparison modes to save the current file.

- Enable the visual indicator (arrow) in the code comparison library, facilitating easy merging of specific sections.

- Introduce a header in "Compare with Original" mode to distinguish between the original file and the current file.

- Update "Compare to Original" to "Compare with Original."

- Rectify the Icon library icon, ensuring it opens in a new tab.

- Incorporate an active indicator in the file list and history file sections of all editors.
